src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,113 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double GROW_COEFFICIENT = 0.5;
+    private static final byte DEFAULT_SIZE = 10;
```

Comment:
```suggestion
    private static final int DEFAULT_SIZE = 10;
```
Let’s  use int (avoid casting)

Reply:
I don't understand, it's a constat, so it will never be neither changed nor casted.